go/sample/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	hk := flag.String("hyperkit", "", "HyperKit binary to use")
 	statedir := flag.String("state", "", "Directory to keep state in")
 	vpnkitsock := flag.String("vpnkitsock", "auto", "Path to VPNKit socket")
-	disk := flag.String("disk", "", "Path to disk image")
+	disks := flag.String("disk", "", "Comma separated list of disk images, format [image]{:size_in_MB}. If size is specified the image will be created if it doesn't exist. If image name is omitted a default name will be used.")
```

Comment:
Yeah I just added something simple in the sample. But it's nice to be consistent - I'll update